nmigen_boards/de0cv.py

```diff
@@ -0,0 +1,107 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.altera import *
+from .resources import *
+
+
+__all__ = ["DE0CVPlatform"]
+
+
+class DE0CVPlatform(AlteraPlatform):
+    device      = "5CEBA4" # Cyclone V 49K LEs
+    package     = "F23"    # FBGA-484
+    speed       = "C7"
+    default_clk = "clk50"
+    resources   = [
+        Resource("clk50", 0, Pins("M9", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("clk50", 1, Pins("H13", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("clk50", 2, Pins("E10", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
+        Resource("clk50", 3, Pins("V15", dir="i"),
+                 Clock(50e6), Attrs(io_standard="3.3-V LVTTL")),
```

Comment:
Wasn't there some sort of global reset pin?

Reply:
P22 (`DEV_CLRN`) is connected to a button to reset the FPGA. To me it doesn't make much sense to add that as a device when the world ends if it's pressed.

Reply:
Is that pin hardcoded in the toolchain? Or is it actually usable in user designs?

Reply:
According to [the pinout list](https://www.intel.com/content/dam/www/programmable/us/en/pdfs/literature/dp/cyclone-v/5ceba4.pdf) it's fixed as `DEV_CLRn`, so I would imagine it's not usable in designs.

Reply:
I think it's usable in user designs:

> You can also tie the DEV_CLRn pin to VCCIO or leave the DEV_CLRn pin unconnected provided that the Enable device-wide reset (DEV_CLRn) option is disabled and not used as a user I/O pin

Reply:
ECP5 has a similar "fixed function" pin which is actually totally arbitrary and just routed like any other global input.

But I think this doesn't really matter, I'll merge as it is since it looks fine.